Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.cpp

```diff
@@ -189,6 +189,12 @@ void RemoteLayerTreeEventDispatcher::handleWheelEvent(const WebWheelEvent& wheel
 {
     ASSERT(isMainRunLoop());
 
+    auto scrollingTree = this->scrollingTree();
+    if (scrollingTree->scrollingPerformanceTestingEnabled()) {
+        if (wheelEvent.phase() == WebWheelEvent::PhaseBegan)
+        WTFBeginSignpostAlways(nullptr, ScrollingPerformanceTestScrollInterval, "isAnimation=YES;");
```

Comment:
Alignment seems off here

Reply:
Thank you

Reply:
Still wrong; it needs to be indented.

Reply:
I'm not convinced that you call  `WTFEndSignpostAlways(..., ScrollingPerformanceTestScrollInterval, ...)` for gestures that don't involve rubberbanding or momentum.

Reply:
Thanks. Working now on confirming if the interval ends without rubber banding or momentum